boot/linux-x86_64/kernel.ld

```diff
@@ -0,0 +1,159 @@
+/*------------------------------------------------------------------------------
+ | Copyright (C) 2018-2019 Daedalean AG
+ |
+ | This library is free software; you can redistribute it and/or modify it
+ | under the terms of the GNU General Public License as published by the
+ | Free Software Foundation; either version 3, or (at your option)
+ | any later version.
+ |
+ | This library is distributed in the hope that it will be useful,
+ | but WITHOUT ANY WARRANTY; without even the implied warranty of
+ | MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ | GNU General Public License for more details.
+ |
+ | Under Section 7 of GPL version 3, you are granted additional
+ | permissions described in the GCC Runtime Library Exception, version
+ | 3.1, as published by the Free Software Foundation.
+ |
+ | You should have received a copy of the GNU General Public License and
+ | a copy of the GCC Runtime Library Exception along with this program;
+ | see the files LICENSE and LICENSE.RUNTIME respectively.  If not, see
+ | <http://www.gnu.org/licenses/>.
+ |----------------------------------------------------------------------------*/
+
+ENTRY(_start)
+
+SECTIONS
+{
+  /*----------------------------------------------------------------------------
+   | 1MB both VMA and LMA - that's where the bootloader will put us.
+   *--------------------------------------------------------------------------*/
+  . = 1M;
+
+  /*----------------------------------------------------------------------------
+   | The text section.
+   |
+   | We need to put the multiboot header first because the bootloader will
```

Comment:
Given that this file is copied and not shared with x86_64, we can remove the multiboot stuff, right?

Reply:
Yeah I agree